addon-test-support/@ember/test-helpers/settled.ts

```diff
@@ -2,6 +2,8 @@
 import { _backburner } from '@ember/runloop';
 import Ember from 'ember';
 
+import EmberApplication from '@ember/application';
```

Comment:
I think this needs to be:

```suggestion
import EmberApplication from '@ember/application/instance';
```